libs/packages/sam-formly/src/lib/formly/wrappers/validation.wrapper.ts

```diff
@@ -1,5 +1,17 @@
 import { Component, ViewChild, ViewContainerRef, OnInit } from '@angular/core';
 import { FieldWrapper } from '@ngx-formly/core';
+/**
+ * @param {string} [to.tagClass] Class to be added to the tag (default: sds-tag--info-white)
+ * @param {string} [to.tagText] Text to be shown inside the tag
+ * @param {string} [to.ariaHidden] Hide the label
+ * @param {string} [to.labelClass] Class to be applied to the label
+ * @param {string} [to.label] Text to be shown for the label
+ * @param {string} [to.required] Makes the field required
+ * @param {string} [to.description] Add a description below the label
+ * @param {string} [to.hideOptional] Remove the optional text
+ * @param {string} [to.hideLabel] Hide the label
```

Comment:
We should only show the template options available in each wrapper, in case they are not using the regular 4 wrappers all together. (same change needed for all wrappers)

Reply:
Updated in all wrappers